app/graph/types/project_media_type.rb

```diff
@@ -85,9 +85,14 @@
 
   field :annotations_count do
     type types.Int
+    argument :annotation_type, types.String
 
-    resolve ->(project_media, _args, _ctx) {
-      project_media.get_versions_log_count
+    resolve ->(project_media, args, _ctx) {
```

Comment:
There's a inconcistency here with 

```connection :annotations, -> { AnnotationType.connection_type } do
    argument :annotation_type, !types.String

    resolve ->(project_media, args, _ctx) {
      project_media.get_annotations(args['annotation_type'].split(',').map(&:strip))
    }
  end
``` 
which does NOT handle blank `annotation_type` BUT handles multiple types separated by commas.